README.md

```diff
@@ -175,13 +186,17 @@ This code sample returns rewards for an Ethereum validator address. View the ful
 // examples/ethereum/list-rewards.ts
 import { StakingClient } from "@coinbase/staking-client-library-ts";
 
+// Set your api key name and private key here. Get your keys from here: https://portal.cdp.coinbase.com/access/api
+const apiKeyName: string = 'your-api-key-name';
```

Comment:
what would be really cool and nice to have is a "test" API key that perhaps has limited scopes etc that I can plaster in all examples. Could be easy for someone wanting to just run the example as is, without going to the platform.